hr_timesheet_employee_analytic_tag/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,10 @@
+To configure this module, you need to:
+
+#. Go to *Settings > Users & Companies > Users*.
+#. Put to your user the following permissions to see all the required elements:
+    * Analytic Accounting
+    * Analytic Accounting Tags
+    * Show Full Accounting Features
+
+#. Go to *Employees > Employees* and edit your employee.
```

Comment:
#. Go to the *Employees* app and edit your employee (should match your user).
#. Choose or create 'Analytic Tags' in 'Timesheets' under 'HR Settings' tab.